### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by tobiii910** [Forceu/barcodebuddy#251] [General]
Is it possible to add German as language? I want to integrate Barcode Buddy to my home, but my family needs everything in German. 

I can do it by myself if someone can explain to me how.

**Comment by Forceu** [comment]
Hi, Barcode Buddy is actually almost completely translated into German (I could translate the missing phrases), but it is currently not possible to apply the translation, as it requires a lot of rewriting. Unfortunately I am not sure, when I will have the time to implement thisl
